src/sentry/integrations/bitbucket/client.py

```diff
@@ -62,7 +65,12 @@ def request(self, method, path, data=None, params=None, **kwargs):
         headers = {
             'Authorization': 'JWT %s' % encoded_jwt
         }
-        return self._request(method, path, data=data, params=params, headers=headers, **kwargs)
+        try:
+            return self._request(method, path, data=data, params=params, headers=headers, **kwargs)
+        except ApiError as e:
+            if 'no issue tracker' in six.text_type(e):
+                pass  # TODO(lb): what do I actually do here?
```

Comment:
Should this error only be ignored in the create, get and search issue operations? Skipping an error for commits because of the message could mask problems.

As for what to do, log and raise an error that let's the user know their request can't be completed.

Reply:
`IntegrationError` doesn't show up as a sentry issue.